### ISSUE TITLE
Celenegil and TiCon

### ISSUE THREAD

**Issue by Durpady** [Davoleo/Metallurgy-4-Reforged#125]
In Metallurgy 4, Celenegil had an enchantability rating of a whopping *fifty* (compared to Gold's 22-25 in vanilla Minecraft).  I feel like its TiCon compatibility should reflect that.  My proposal:

As a head material, all modifiers are treated as *two levels higher* than the materials that went into them.  This would allow the tool to exceed the usual cap, i.e., Sharpness VII and Looting V.  As a non-head material, it would grant *+2 modifiers*, though this wouldn't stack.

**Comment by Davoleo** [comment]
> As a head material, all modifiers are treated as two levels higher than the materials that went into them. This would allow the tool to exceed the usual cap, i.e., Sharpness VII and Looting V. As a non-head material, it would grant +2 modifiers, though this wouldn't stack.

I didn't quite understand your proposal, with "two levels higher" are you still talking about enchantability? if yes how much is 2 levels of enchantability?

And, concerning non-head parts, I don't get the link between enchantability and modifiers, could you explain it a bit more?

**Comment by HurtsAsHell** [comment]
Not having received an answer after 9 days, I close the issues, if you want to add more, go ahead but continue below

**Comment by Durpady** [comment]
Whoops, didn't mean to leave this for so long.

I mean that its TiCon aspects should _reflect_ its enchantability level.  By "two levels higher", I meant that an item that's been given, say, enough Consecrated Soil to normally be at Smite I would instead be at Smite III; when it's been given enough to be at Smite IV, it would instead be at Smite VI.

That is, if y'all can actually implement that with TiCon 2's current code structure.

**Comment by Davoleo** [comment]
This could make the whole tinker tools very OP
I'm not sure if we want to push people even more to use our mod just as some sort of Tinkers' Construct Addon by making these tools even more powerful 
I hope you get what I'm thinking, and sorry for the late response @Durpady

**Comment by Durpady** [comment]
Alright, so, math time.  I am _assuming_ here that TiCon's internal modifier mechanics math follows vanilla Minecraft's enchanting math mechanics.

Assuming the base damage values and Sharpness damage bonus values are the same, Tartarite + Sharpness V does exactly the same damage as Celenegil + Sharpness VII.  Smite and BoA would do more damage, but they're still constrained by the classic problem of not being compatible with each other or Sharpness.

Fortune V... I went into the game with some command'd in Diamond Pickaxes, and smashed 64 Coal Ore blocks each with Efficiency VII / Fortune V and Efficiency V / Fortune III.  The "regular max" produced 15 blocks of coal, while the "supercharged" one produced 23 blocks.  Let's call this a 50% increase of yield over Fortune III.

Knockback is honestly a liability to me.  Knockback IV seems like it'd smack things back _so_ far that its usefulness would only be situational.

Looting V would give an 11% chance of _not_ receiving an Ender Pearl, compared to a 17% chance with Looting III.  That's approximately a reduction in "failure" by 50%.  _Maximum_ yield for, say, porkchops or bones, would increase from 4 to 6, so the median yield (let's say it's the average yield) goes up from 2.5 to 3.5.  Meanwhile rare drops (Rabbit's Foot?) go up to about 1 in 12 from 1 in 20, while nabbing an equipment piece from a mob goes up from about 1 in 8 to about 1 in 7.  These are some pretty notable increases.

Fire Aspect IV would deal... 7 1/2 Hearts of damage.  You got me, that's a _lot_.  Only thing I can say about that is that it'd take a bit for the full damage to unfold (which is sub-optimal in a combat environment), wouldn't work in rain/water, and wouldn't work on nether mobs.

Efficiency VII adds 50 to the tool mining speed; Efficiency V adds 26.  This seems like it would harvest everything other than Obsidian instantly (though I don't think the difference between 0.5s and 0.2s is really a game-changer.  Obsidian seems to go down from a little over 2s to about 1.5s.  Honestly this is a QoL improvement for harvesting Obsidian more than it is anything else.  In fact it could even run into the Netherrack problem of harvesting things _too_ quickly.  Remember that test I did earlier?  I guess VII was noticeably faster, but that was a difference I could easily live without, further cementing in my mind it's nothing game-changing.

It looks like the biggest problems introduced by this are Fortune/Looting and Fire Aspect, _maybe_ efficiency.

So, your argument has merit to it, but I'm betting it's a bit less than you were thinking.  This sounds more like "ultimate mining tool head" than "bonkers weapon head", which, frankly, surprises me.
